src/test/java/pages/UploadPage.java

```diff
@@ -16,17 +21,23 @@ public UploadPage(WebDriver driver) {
     public static final By UPLOADED_DIV = By.xpath("//*[@id='uploaded-files']");
 
     public void openPage() {
-        String url = BasePageConstants.BASE_URL + UploadPageConstants.URL;
-        driver.get(url);
+        driver.get(BASE_URL + URL);
     }
 
-    public void uploadFile(){
+    public void uploadFile() {
         WebElement upload = driver.findElement(UPLOAD_FILE_INPUT);
-        upload.sendKeys("\\src\\test\\resources\\TestUpload.txt");
+        URL pathURL = getClass().getClassLoader().getResource("TestUpload.txt");
```

Comment:
try to avoid hardcode values such as "TestUpload.txt", put them as parameter in method and pass necessary value in Test classes. Please fix it.

Reply:
done